packages/server/lib/modes/run.js

```diff
@@ -647,6 +647,24 @@ const getVideoRecordingDelay = function (startedVideoCapture) {
   return 0
 }
 
+const generateFfmpegChaptersConfig = (testResults) => {
```

Comment:
`run.js` is intended for logic for `cypress run`, so this video functionality should go in a different file. `video_capture.coffee` would probably be a better place for the logic in this function.